README.md

```diff
@@ -33,3 +33,37 @@ Example:
   "head_sha": "9130a5840e1ac7ffe198795ba663fdd545b390c3"
 }
 ```
+
+## Staging rollout procedure
+
+To rollout to staging, we want to first make staging be identical with prod. Thus, we clone the prod db to staging, then restart the backend on the cloned db:
```

Comment:
This is quite a heavy duty workflow. In my view it makes sense when there are changes to the backend/preprocessing/ingest (especially if substantial, but perhaps in any case). When we are just adding some new website features/tweaks in my view it is sufficient that there is a staging deployment at the same commit as current production, with data that comes from a clone of production at some point in the past.

Reply:
(just clarifying whether this is a matter of documenting a useful procedure or creating a policy)

Reply:
I'm just describing the current practice here that @anna-parker and I have been doing. It also doesn't take that long, especially with the commands documented. If you want to take the risk (and responsibility) of skipping the clone you can do so, but I think it's good to state the ideal here.

Reply:
👍 I really like the ability to deploy stuff without opening up a terminal (and so constructed all the GitHub workflows etc. around that) so would be glad to maintain a policy that allows us to do that at least for website changes. Maybe sometime later we will be inspired to make the cloning etc. part of that, which would be cool

Reply:
The integrity stuff could be turned into an action - I pretty much live inside terminals so for me the terminal thing is not an issue at all but I can see that that's not universal

Just now it caught this which is nice to see:

```
1 row changed

  submissionId: CL292200
    hostAge: "32" => "32.0"
```